query/ebs/ebs_attached_volume_encryption_enabled.sql

```diff
@@ -2,17 +2,17 @@ select
   -- Required Columns
   arn as resource,
   case
+    when state != 'in-use' then 'info'
```

Comment:
```suggestion
    when state != 'in-use' then 'skip'
```

If we're not evaluating them, should it be in `skip` state instead? What does the original query/config rule say about evaluating unattached volumes (or doesn't say)?

Reply:
@cbruno10 this is what it says below; non-attached ones we can `skip`

```
This control checks whether the EBS volumes that are in an attached state are encrypted. To pass this check, EBS volumes must be in use and encrypted. If the EBS volume is not attached, then it is not subject to this check.
```